KrampitzKreutzwiserICS4U/src/krampitzkreutzwiserics4u/MaterialReview.java

```diff
@@ -128,8 +128,8 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
      */
     private void backBtnActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_backBtnActionPerformed
         // TODO add your handling code here:
-        mainMenuFrame.setVisible(true);
         this.setVisible(false);
+        mainMenuFrame.setVisible(true);
```

Comment:
I just changed the order because I think it will be better if there ever is a stutter or something then there won't ever be two windows visible at a time